portfolio/src/main/java/com/google/sps/ServerMain.java

```diff
@@ -15,30 +15,28 @@
  */
 public class ServerMain {
 
-  public static void main(String[] args) throws Exception {
-
-    // Create a server that listens on port 8080.
-    Server server = new Server(8080);
-    WebAppContext webAppContext = new WebAppContext();
-    server.setHandler(webAppContext);
-
-    // Load static content from inside the jar file.
-    URL webAppDir = ServerMain.class.getClassLoader().getResource("META-INF/resources");
-    webAppContext.setResourceBase(webAppDir.toURI().toString());
-
-    // Enable annotations so the server sees classes annotated with @WebServlet.
-    webAppContext.setConfigurations(
-        new Configuration[] {
-          new AnnotationConfiguration(), new WebInfConfiguration(),
-        });
-
-    // Look for annotations in the classes directory (dev server) and in the jar file (live server)
-    webAppContext.setAttribute(
-        "org.eclipse.jetty.server.webapp.ContainerIncludeJarPattern",
-        ".*/target/classes/|.*\\.jar");
-
-    // Handle static resources, e.g. html files.
-    ServletHolder defaultServletHolder = webAppContext.addServlet(DefaultServlet.class, "/");
+  public static void main(final String[] args) throws Exception {
+
+        // Create a server that listens on port 8080.
+        final Server server = new Server(8080);
+        final WebAppContext webAppContext = new WebAppContext();
+        server.setHandler(webAppContext);
+
+        // Load static content from inside the jar file.
+        final URL webAppDir = ServerMain.class.getClassLoader().getResource("META-INF/resources");
+        webAppContext.setResourceBase(webAppDir.toURI().toString());
+
+        // Enable annotations so the server sees classes annotated with @WebServlet.
+        webAppContext
+                .setConfigurations(new Configuration[] { new AnnotationConfiguration(), new WebInfConfiguration(), });
+
+        // Look for annotations in the classes directory (dev server) and in the jar
+        // file (live server)
+        webAppContext.setAttribute("org.eclipse.jetty.server.webapp.ContainerIncludeJarPattern",
+                ".*/target/classes/|.*\\.jar");
+
+        // Handle static resources, e.g. html files.
+        final ServletHolder defaultServletHolder = webAppContext.addServlet(DefaultServlet.class, "/");
```

Comment:
As far as I can tell there are no changes to the code here, only extra indentation. We want consistent style (e.g. indentation throughout the code base). Please fix. 

This file follows the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html) which says

> "Each time a new block or block-like construct is opened, the indent increases by two spaces." - [source](https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation)

In real life, you're development tools will automatically apply a style guide for you, which is a big help and means this typically doesn't show up in code reviews. But, we're working on Cloud Shell which has limited dev tools. No biggie!